obstacle_stop_planner_refine/include/obstacle_stop_planner/adaptive_cruise_control.hpp

```diff
@@ -43,8 +43,8 @@ class AdaptiveCruiseController
     const rclcpp::Time nearest_collision_point_time,
     const autoware_perception_msgs::msg::DynamicObjectArray::ConstSharedPtr object_ptr,
     const geometry_msgs::msg::TwistStamped::ConstSharedPtr current_velocity_ptr,
-    bool * need_to_stop,
-    autoware_planning_msgs::msg::Trajectory * output_trajectory);
+    bool & need_to_stop,
+    autoware_planning_msgs::msg::Trajectory & output_trajectory);
```

Comment:
I'd like to define a struct and make these output values the return value (in the later stage of this refactoring).